stonesoup/updater/kalman.py

```diff
@@ -222,7 +222,7 @@ def predict_measurement(self, predicted_state, measurement_model=None,
 
         # The measurement cross covariance and innovation covariance
         meas_cross_cov = self._measurement_cross_covariance(predicted_state, hh)
-        innov_cov = self._innovation_covariance(meas_cross_cov, hh, measurement_model)
+        innov_cov = self._innovation_covariance(meas_cross_cov, hh, measurement_model, **kwargs)
```

Comment:
@spike-dstl I think this the line that is causing the docs to fail

Reply:
I have made changes so that `BayesianRecursiveUpdater`'s `_innovation_covariance()` method calls from properties of the class rather than using **kwargs. Looks like this has solved the issue, thanks for spotting it.